portality/scripts/2882_empty_string_issns.py

```diff
@@ -0,0 +1,52 @@
+from portality import models
+from portality.core import es_connection
+import esprit
+import csv
+from portality.util import ipt_prefix
+
+EMPTY_ISSN = {
+"query": {
+        "filtered": {
+            "filter": {
+                "bool": {
+                    "must": {
+                        "term": {"admin.in_doaj": "true"}
+                    }
+                }
+            },
+            "query": {
+                "match_all": {}
+            }
+        }
+    }
+}
+
+if __name__ == "__main__":
```

Comment:
We could probably run this on test to get an idea of the answer, without merging and deploying to live first.

Reply:
Agree.

Reply:
Locally does not return anything.

Reply:
```

(doaj) cloo@doaj-new-test-1:~/doaj/src/doaj$ DOAJENV=test python portality/scripts/2882_empty_string_issns.py 
Running in test
Loaded environment config from /home/cloo/doaj/src/doaj/test.cfg
Loaded secrets config from /home/cloo/doaj/src/doaj/app.cfg
Compiling static content
Compiling SASS
Analyzed 15870 journals in DOAJ
```

Reply:
`out.csv` empty so nothing stopping the release and run on live.